R/correlation.R

```diff
@@ -27,6 +27,26 @@ Correlation <- function(jaspResults, dataset, options){
   return()
 }
 
+.corrGetTests <- function(options){
+  tests <- c("pearson", "spearman", "kendall")
+  testsNames <- c(gettext("Pearson's"), gettext("Spearman's"), gettext("Kendall's Tau"))
+  
+  whichTests <- c(options[['pearson']], options[['spearman']], options[['kendallTauB']])
+  usedTests <- tests[whichTests]
+  usedTestsNames <- testsNames[whichTests]
+  
+  return(list(
+    tests = tests, testsNames = testsNames,
+    usedTests = usedTests, usedTestsNames = usedTestsNames
+  ))
+}
+
+.corrTestChecked <- function(test = c('pearson', 'spearman', 'kendallsTauB'), options){
```

Comment:
Nice hack. Another way to solve this however is to change the update json so "kendallsTauB" is changed to "kendall", which is more inline with what cor.test takes as an argument for test

Reply:
do you mean qml or json? I can rename the option, but I was not sure whether this renaming of things in the qml needs some thinking about backward compatibility... Or is that not an issue?

Reply:
see https://github.com/jasp-stats/Regression/pull/1#discussion_r431720510